src/main/app/case/CosApiClient.ts

```diff
@@ -0,0 +1,33 @@
+import Axios, { AxiosInstance } from 'axios';
+import config from 'config';
+
+import { getServiceAuthToken } from '../auth/service/get-service-auth-token';
+import type { AppSession } from '../controller/AppRequest';
+
+export class CosApiClient {
+  client: AxiosInstance;
+
+  constructor(private readonly session: AppSession, readonly returnUrl: string) {
+    this.client = Axios.create({
+      baseURL: config.get('services.cos.url'),
+      headers: {
+        Authorization: 'Bearer ' + session.user.accessToken,
+        serviceAuthorization: getServiceAuthToken(),
+        'return-url': returnUrl,
+      },
+    });
+  }
+
+  public async get(): Promise<string | undefined> {
+    try {
+      const response = await this.client.get<string>('/');
+      const userCase = this.session.userCase;
```

Comment:
userCase is not being used here except in console log. Please see if this can be removed.